spec/controllers/state_file/questions/md_social_security_benefits_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+RSpec.describe StateFile::Questions::MdSocialSecurityBenefitsController do
+  let(:intake) { create :state_file_md_intake }
+  before do
+    sign_in intake
+  end
+
+  describe ".show?" do
+    context "when the client has fed ssb but is not married filed jointly" do
+      let(:intake) { create :state_file_md_intake, :df_data_many_w2s }
+      it "returns false" do
+        expect(described_class.show?(intake)).to eq false
+      end
+    end
+
+    context "when the client is mfj" do
+      context "but does not have fed ssb" do
+        let(:intake) { create :state_file_md_intake, filing_status: "married_filing_jointly" }
+        it "returns false" do
+          expect(described_class.show?(intake)).to eq false
+        end
+      end
+
+      context "and has fed ssb in the fed return" do
+        let(:intake) { create :state_file_md_intake, :df_data_many_w2s, filing_status: "married_filing_jointly" }
+        it "returns true" do
+          expect(described_class.show?(intake)).to eq true
+        end
+      end
+    end
+  end
+
+
+  describe "#edit" do
```

Comment:
🤔 this is more for my understanding, but how come you don't need to ensure that the .show? method allows for the #edit action to be seen.